VideoLocker/src/main/java/org/edx/mobile/base/MainApplication.java

```diff
@@ -171,9 +173,10 @@ private void tryToUpdateParseForAppUpgrade(Context context){
         if (  previousVersion < curVersion ){
             ParseInstallation installation = ParseInstallation.getCurrentInstallation();
 
-            String  android_id = Secure.getString(getApplicationContext().getContentResolver(), Secure.ANDROID_ID);
-            installation.put("UniqueId",android_id);
+            String language = Locale.getDefault().getLanguage();
+            installation.put("preferredLanguage",language);
```

Comment:
After a little research, I think that iOS and Android are using different versions of the language code standards. We should circle back and make sure we're sending the same things. Also, I think we may be confusing *language* and *locale*